spec/models/user_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  subject { User.new(name: nil, bio: 'Student from Argentina', posts_counter: nil) }
+
+  before { subject.save }
+
+  it 'Name should be present' do
+    expect(subject).to_not be_valid
+  end
+
+  it 'No Post counter should raise error' do
+    expect(subject).to_not be_valid
+  end
```

Comment:
- [ ] Let's test the posts counter just to be sure when a new post is created, the counter will be increased. Do not forget to test for the positive side of the valid attributes, such as if a name is provided.